src/config.ts

```diff
@@ -36,7 +36,7 @@ export class ExecutorConfig {
   readonly package?: Package | string;
   readonly maxParallelTasks: number;
   readonly taskTimeout: number;
-  readonly startupTaskTimeout: number;
```

Comment:
In one way that's a BC-breaking change (removal of the field). We will have to point this out in the release notes. Probably that will bump the version of the TE :D

Reply:
I don't think so, this is just a change in the `config` file. The public `ExecutorOptions` remain unchanged as the correct `taskStartupTimeout`.